dpc/src/record/payload.rs

```diff
@@ -18,19 +18,21 @@ use snarkvm_utilities::{FromBytes, ToBytes};
 
 use std::io::{Read, Result as IoResult, Write};
 
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Default)]
-pub struct Payload([u8; 32]);
+pub const PAYLOAD_SIZE: usize = 128;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Payload([u8; PAYLOAD_SIZE]);
 
 impl Payload {
     pub fn to_bytes(&self) -> &[u8] {
         &self.0[..]
     }
 
     pub fn from_bytes(bytes: &[u8]) -> Self {
-        assert_eq!(bytes.len(), 32);
+        assert_eq!(bytes.len(), PAYLOAD_SIZE);
 
-        let mut payload = [0u8; 32];
-        payload.copy_from_slice(&bytes[0..32]);
+        let mut payload = [0u8; 128];
+        payload.copy_from_slice(&bytes[0..PAYLOAD_SIZE]);
```

Comment:
```suggestion
        payload.copy_from_slice(bytes);
```
`bytes` is already a slice and it's guaranteed to be of `PAYLOAD_SIZE` due to the use of `copy_from_slice` and `payload` being of that size